vita3k/util/src/vc_runtime_checker.cpp

```diff
@@ -26,7 +26,7 @@ struct VCRuntimeCheckObject {
 
         const HANDLE heap = GetProcessHeap();
         DWORD filename_length = MAX_PATH;
-        LPWSTR filename = static_cast<LPWSTR>(HeapAlloc(heap, 0, filename_length));
+        LPWSTR filename = static_cast<LPWSTR>(HeapAlloc(heap, 0, filename_length * sizeof(*filename)));
```

Comment:
Nice catch! PCSX2 and Duckstation are using the same code, we should inform them about it.

Reply:
Also, although it works, using sizeof(*filename) before the variable filename gets initialized is a bit weird.

Reply:
I have no idea, how to get this value `sizeof(*filename)` `sizeof(LPWSTR)` return size of the pointer, not a single character under this pointer. (Although they are the same)